src/hubspot/index.ts

```diff
@@ -0,0 +1,51 @@
+import fetch, { RequestInit } from 'node-fetch';
+import qs from 'qs';
+
+export default class Hubspot {
+  constructor(
+    private readonly apiBaseUrl: string,
+    private readonly hapikey: string,
```

Comment:
Not :100: sure if this is typo or you wanted to take 'h' from 'HubSpot', but I'd just call it apiKey, especially considering we call the other property `apiBaseUrl` :slightly_smiling_face: